code/datums/diseases/advance/symptoms/feline.dm

```diff
@@ -0,0 +1,60 @@
+//I am actually going insane from a second project so I am writing a meme symptom for an uplink item to avoid burnout.
+/datum/symptom/feline
+	name = "Feline Minor Retrovirus"
+	desc = "A derivative of the fabled felinid virus"
+	stealth = -2
+	resistance = -2
+	stage_speed = 1
+	transmittable = 2
+	level = -1 //Uplink Symptom
+	severity = 3 //Class F Biohazard
+	symptom_delay_max = 20
+	symptom_delay_max = 60
+	var/nya_transmit = FALSE //Var that stores the transmission threshhold
+	var/full_feline = FALSE //Var that checks the stage speed and resistance threshold
+	threshold_desc = "<b>Transmission 12:</b> Chance for virus to spread when nyaaa ing. <br>\
+	<b>Stage Speed 12 and Resistance 10:</b> Will mutate the species into felinid."
+
+/datum/symptom/feline/severityset(datum/disease/advance/A)
+	if(A.properties["transmittable"] >= 12)
+		severity+= 1
+	if(A.properties["stage_speed"] >= 12 && A.properties["resistance"] >= 10)
+		severity+= 99 //Sets Severity to a Custom Felinid Virus Biohazard Classification
+	..()
+
+/datum/symptom/feline/Start(datum/disease/advance/A)
+	if(!..())
+		return
+	if(A.properties["transmittable"] >= 12)
+		nya_transmit = TRUE
+	if(A.properties["stage_speed"] >= 12 && A.properties["resistance"] >= 10)
+		full_feline = TRUE //OH NYOOO WE NEED TO LYNCH THE VIWOLOGIST
+
+/datum/symptom/feline/Activate(datum/disease/advance/A)
+	if(!..())
+		return
+	var/mob/M = A.affected_mob
+	switch(A.stage)
+		if(2)
+			if(prob(40))
+				M.say(pick("Nya!", "Nya~", "~meow~", "N-nya"))
```

Comment:
Kill me

Reply:
Yes

Reply:
nyannyan uwu

Reply:
Not sure what to do with the review here.